src/bridge/settings/openedx/mfe/slot_config/SidebarAIDrawerCoordinator.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useContext, useEffect, useState, useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { getConfig } from '@edx/frontend-platform';
+import { useModel } from '@src/generic/model-store';
+
+import Sidebar from './src/courseware/course/sidebar/Sidebar';
+import NewSidebar from './src/courseware/course/new-sidebar/Sidebar';
+import SidebarContext from './src/courseware/course/sidebar/SidebarContext';
+import NewSidebarContext from './src/courseware/course/new-sidebar/SidebarContext';
+import AIDrawerManagerSidebar from './AIDrawerManagerSidebar';
+
+const AI_DRAWER_MESSAGE_TYPES = [
+    'smoot-design::ai-drawer-open',
+    'smoot-design::tutor-drawer-open',
+];
+
+const SidebarAIDrawerCoordinator = ({ courseId }) => {
+    const {
+        isNewDiscussionSidebarViewEnabled,
+    } = useModel('courseHomeMeta', courseId);
+
+    const oldContextValue = useContext(SidebarContext);
+    const newContextValue = useContext(NewSidebarContext);
+    const contextValue = isNewDiscussionSidebarViewEnabled ? newContextValue : oldContextValue;
+    const currentSidebar = contextValue?.currentSidebar ?? null;
+    const toggleSidebar = contextValue?.toggleSidebar ?? (() => {});
+    const shouldDisplayFullScreen = contextValue?.shouldDisplayFullScreen ?? false;
+
+    const [showAiDrawer, setShowAiDrawer] = useState(false);
```

Comment:
I would like to be a bit more consistent with the variable/component naming. We should use capital case `i` here and all other places.
```suggestion
    const [showAIDrawer, setShowAIDrawer] = useState(false);
```